2-mandatory/1-writers.js

```diff
@@ -71,19 +71,25 @@ Exercise 2:
 */
 
 function logDeadWritersInTheirForties() {
-  // write your code here
+   writers.filter(x => x.age >= 40 && x.age <= 49 && x.alive === false)
```

Comment:
Hi Jihad, nice work. We can understand what x does in the function but what if you're trying to explain the code to a total beginner? I suggest using a naming that can be more understandable to people of all levels, to understand.

Reply:
> Hi Jihad, nice work. We can understand what x does in the function but what if you're trying to explain the code to a total beginner? I suggest using a naming that can be more understandable to people of all levels, to understand.

Hi Levi, Thank you for your review, I think you are correct.

Reply:
> Hi Jihad, nice work. We can understand what x does in the function but what if you're trying to explain the code to a total beginner? I suggest using a naming that can be more understandable to people of all levels, to understand.

I completely agree with you Levi

Reply:
I agree with you, Thank you for your review @Shimaadnan